Source/MLXRandom/Cmlx+Util.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2024 Apple Inc.
+
+import Cmlx
+import Foundation
+import MLX
```

Comment:
Needed this file in MLXRandom as we have a function that returns a tuple.  Ideally I would like this to be in MLX and exposed only to the local libraries, but I think I have to have a private library to pull it off.  In the meantime we have a couple copies of this file.